cmake/alpakaCommon.cmake

```diff
@@ -368,6 +369,16 @@ if(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 message(FATAL_ERROR "Clang as a CUDA compiler does not support OpenMP 5!")
             endif()
 
+            if(ALPAKA_RELOCATABLE_DEVICE_CODE)
+                if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 13.0)
```

Comment:
Is this the workaround for https://github.com/alpaka-group/alpaka/issues/1292#issuecomment-823097652?